compiler/rustc_builtin_macros/src/format.rs

```diff
@@ -186,6 +186,8 @@ fn make_format_args(
                             ],
                              Applicability::MaybeIncorrect,
                         );
+                    } else if unexpanded_fmt_span.in_proc_macro_expansion() {
+                        err.help("you might be missing a string literal to format with in the definition of this macro");
```

Comment:
A lot of compiler suggestions are skipped for code from macros, but suggestions to look into the macro like this are typically not emitted in such cases.